modules/system.js

```diff
@@ -56,6 +56,7 @@ class System {
 				nethash: scope.config.nethash,
 				minVersion: scope.config.minVersion,
 				nonce: scope.config.nonce,
+				blackListPeers: scope.config.blackListPeers,
```

Comment:
Why do we need this list in `system` module. Its responsibility of `peers` module and only kept there.